source/_patterns/03-templates/page-grid/page-grid.twig

```diff
@@ -0,0 +1,33 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['page-grid']) }) -%}
+
+<div {%- include 'atoms-attributes' %}>
+
+  {%- with {items: items} only -%}
+    {%- block items -%}
+
+      {%- for item in items -%}
+
+        {%- with item only -%}
+          {%- block item -%}
+
+            {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['page-grid-item']) }) -%}
+
+            {%- if keylineStart|default(false) -%}
+              {%- set attributes = attributes|merge({class: attributes.class|merge(['page-grid-item--has-keyline-start']) }) -%}
+            {%- endif -%}
+
+            {%- if keylineEnd|default(false) -%}
+              {%- set attributes = attributes|merge({class: attributes.class|merge(['page-grid-item--has-keyline-end']) }) -%}
+            {%- endif -%}
+
+            {%- include template|default('templates-content-grid') -%}
```

Comment:
Default should be a single column grid?

Reply:
If there is only `primaryContent`, then the `content-grid` template effectively is a single column template. Do you think we need an additional template that is  explicitly only ever a single column? Feels like duplication, but happy to be convinced otherwise if it's necessary 😄

Reply:
There's two types of single-column grids though, `full`-width and `main`-width. This covers the latter. (The content headers on the sample pages don't currently work, changing `content` to `primaryContent` there sees them squashed into the `main`.)

Reply:
(That's what https://github.com/libero/pattern-library/pull/54#discussion_r260219881 refers to.)